__tests__/core/utils/NetworkConfigurationHelpers.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { NetworkType } from 'symbol-sdk';
+import { OfflineNetworkProperties } from '@/services/offline/MockModels';
+import { NetworkConfigurationHelpers } from '@/core/utils/NetworkConfigurationHelpers';
+import { networkConfig } from '@/config';
+
+describe('core/utils/NetworkConfigurationHelpers', () => {
+    describe('epochAdjustment()', () => {
```

Comment:
please add the command

Arrange, Act and Assert

Reply:
added :)